content/docs/overview.md

```diff
@@ -1311,6 +1311,22 @@ The above array can also be constructed with a question mark (`?`) to automatica
 x := [?]int{1, 2, 3, 4, 5}
 ```
 
+Construct an array with designated initializers:
+
+```odin
+favorite_animals := [?]string{
+	0 = "Raven",
+	1 = "Zebra",
+	2 = "Spider",
+	3 = "Frog",
+	4 = "Badger"
+}
```

Comment:
I'd also recommend show ranges working too e.g. `3..=5 = "Dog",` and `5..<9 = "Cat",`.